TappNetwork/home-visiting-app-web
# Cu 3nnbzkf feedback ux tweaks

# Details
This PR incorporates the visual and copy changes requested by the client during their review. Back end copy changes will be bundled into next backend PR.

# Steps to test
1. Go to login page of front end
2. Notice that input fields now have a white background to make their function more clear
4. Login with visitor account 
5. Profile card should now have an edit profile button
6. References to home visitor and visitor should now be absent from app, including urls.
7. Make sure navigation to Family Support Specialist, FSS, directory and FSS profiles is not broken by this change. 

# Tasks Links
[3nnbyp5](https://app.clickup.com/t/3nnbyp5)
[3nnbzkf](https://app.clickup.com/t/3nnbzkf)

# Screenshots
<img width="316" alt="Screen Shot 2022-10-10 at 12 44 38 PM" src="https://user-images.githubusercontent.com/29612767/194932952-32b78448-1b2d-4a08-9d10-7c1a54045986.png">
<img width="1036" alt="Screen Shot 2022-10-10 at 12 44 49 PM" src="https://user-images.githubusercontent.com/29612767/194932959-223ea7e1-f302-47ca-8f79-055dbafece68.png">
